src/client/app/colyseus.game.service.ts

```diff
@@ -213,7 +213,7 @@ export class ColyseusGameService {
     if(context === 'Ground') {
       args = `${item.itemClass} ${item.uuid}`;
 
-    } else if(includes(['Sack', 'Belt', 'Equipment'], context)) {
+    } else if(includes(['Locker', 'Sack', 'Belt', 'Equipment'], context)) {
```

Comment:
Call it a Kubby instead. You can't use Locker as the context because L is Left Hand already.